src/main/environment/common_dev.properties

```diff
@@ -17,6 +17,9 @@ carestreamOrderCreateURL =http://10.208.122.32:8080/hwc-commonapi-v1.0/carestrea
 #registrationUrl =http://localhost:8080/beneficiary/create
 registrationUrl =http://10.208.122.32:8080/hwc-commonapi-v1.0/beneficiary/create
 
+#syncSearchByLocation =http://localhost:8082/id/searchByDistrictId
+syncSearchByLocation =http://10.208.122.32:8082/id/searchByDistrictId
```

Comment:
check this once, it is wrong url i think

Reply:
Change done. You can review again please @roopesh-beehyv @Shrey475